src/devices/Nmea0183/Ais/AisSafetyState.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Nmea0183.Ais
+{
+    /// <summary>
+    /// Defines how safe a given target is in relation to typically our own vessel.
+    /// Safe here means that there's little risk of a collision within the defined safety bounds (e.g. the ship is more than 5 minutes away)
+    /// </summary>
+    public enum AisSafetyState
+    {
+        /// <summary>
+        /// It's not known whether this ship comes close
+        /// </summary>
+        Unknown,
+
+        /// <summary>
+        /// The ship is moving away from us or it is very far away
+        /// </summary>
+        Safe,
+
+        /// <summary>
+        /// There's an imminent risk of collision with this ship
+        /// </summary>
+        Dangerous,
```

Comment:
Should this be InDanger? Danger suggests it's a threat rather than being in danger

Reply:
That's what's meant: That other ship is dagerous to us. A "dangerous AIS target" is one that I have on my screen and is dangerous to me.